src/dials/algorithms/refinement/parameterisation/configure.py

```diff
@@ -80,6 +80,12 @@
       .type = bool
       .short_caption = "Scan-varying refinement"
 
+    interval_width_degrees = None
+        .help = "Overall default value of the width of scan between checkpoints"
+                "in degrees for scan-varying refinement. If set to None, each"
+                "model will use its own specified value."
+        .type = float(value_min=0.)
```

Comment:
Would we want to set a maximum value? 90° or something? Or is that an exercise for the user?